content/java/concepts/strings/strings.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Strings'
-Description: 'A String in Java is an object that holds a sequence of characters contained within a pair of double quotes ("). It is not a primitive datatype. Java strings provide a way to store something like a word, sentence, or whole paragraph. They can be any length and can contain letters, numbers, symbols, and spaces. java // Creating a String variable String name = "Codecademy"; // Creating another String variable String address = "575 Broadway #5, New York, NY 10012"'
+Description: 'A String in Java is an object that holds a sequence of characters contained within a pair of double quotes (").'
```

Comment:
I don't believe string should be capitalized. I understand that in Java code the type is declared as 'String' but we generally discuss items in docs generically: referring to the class of object in lieu of the code. I reviewed the Python and Javascript pages and they both refer to strings generically